<?php

return [

    'update' => [
        'error'                 => 'При изменении произошла ошибка. ',
        'success'               => 'Настройки успешно изменены.',
    ],
    'backup' => [
        'delete_confirm'        => 'Вы уверены, что хотите удалить эту резервную копию? Это действие нельзя отменить. ',
        'file_deleted'          => 'Резервная копия была успешно удалена. ',
        'generated'             => 'Новая резервная копия была успешно создана.',
        'file_not_found'        => 'Эта резервная копия не найдена на сервере.',
        'restore_warning'       => 'Да, восстановить. Я осознаю, что это перезапишет все существующие данные в базе данных. Это также выйдет из учетных записей всех ваших существующих пользователей (включая вас).',
        'restore_confirm'       => 'Вы уверены, что хотите восстановить базу данных из :filename?'
    ],
    'purge' => [
        'error'     => 'Возникла ошибка во время очистки. ',
        'validation_failed'     => 'Ваш текст подтверждения очистки неверен. Пожалуйста, наберите слово "DELETE" в поле подтверждения.',
        'success'               => 'Удалённые записи успешно очищены.',
    ],
    'mail' => [
        'sending' => 'Отправляется тестовое электронное письмо...',
        'success' => 'Письмо отправлено!',
        'error' => 'Не удалось отправить электронное письмо.',
        'additional' => 'Нет дополнительных сообщений об ошибке. Проверьте настройки почты и журнал вашего приложения.'
    ],
    'ldap' => [
        'testing' => 'Тестирование подключения к LDAP, привязка & запрос ...',
        '500' => 'Ошибка в 500 сервере. Пожалуйста, проверьте журналы сервера для получения дополнительной информации.',
        'error' => 'Что-то пошло не так :(',
        'sync_success' => 'Пример 10 пользователей, полученных с сервера LDAP, на основе ваших настроек:',
        'testing_authentication' => 'Тестирование LDAP аутентификации...',
        'authentication_success' => 'Пользователь успешно аутентифицирован с LDAP!'
    ],
    'webhook' => [
        'sending' => 'Отправка тестового сообщения :app...',
        'success_pt1' => 'Успешно! Проверьте ',
        'success_pt2' => ' канал для вашего тестового сообщения и не забудьте нажать Сохранить ниже, чтобы сохранить ваши настройки.',
        '500' => '500 Ошибка сервера.',
        'error' => 'Что-то пошло не так. :app ответил: :error_message',
        'error_misc' => 'Что-то пошло не так. :( ',
    ]
];
